addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -443,7 +443,7 @@ body.editor_enable.editor_has_snippets {
                     margin-bottom: 1px; // Not sure why but not really centered otherwise
                 }
             }
-            &.active {
+            &.active:not([data-shape]) {
```

Comment:
Don't make this something specific to shapes but rather to a class you can add on any select/button